internal/pkg/template/workload.go

```diff
@@ -399,6 +399,12 @@ type Topic struct {
 	Svc       string
 }
 
+// Fifo holds configuration needed if the topic is FIFO.
+type FIFOTopicConfig struct {
+	Enable                    *bool
+	ContentBasedDeduplication *bool
```

Comment:
I wonder what'd happen in this case:
1. First deploy the CFN template with `ContentBasedDeduplication: true` field
2. Then deploy the CFN template without the field at all

Do they set `ContentBasedDeduplication` to `false`, or keep it as `true`?

Reply:
If `ContentBasedDeduplication` is not there in the template then CFN sets it to false by default

Reply:
Then what about we have 
```
ContentBasedDeduplication bool
```

and then in template we have
```
ContentBasedDeduplication: {{ .ContentBasedDeduplication }}
```

If CFN always sets it to `false` if it's not there (instead of keeping the existing config) then this should have the same effect as the current implementation but the logic would be simpler.